tests/rule_based_profiler/config/test_base.py

```diff
@@ -190,11 +189,7 @@ def test_rule_config_unsuccessfully_loads_with_missing_required_fields():
 
     assert all(
         f"'{attr}': ['Missing data for required field.']" in str(e.value)
-        for attr in (
-            "name",
-            "domain_builder",
-            "expectation_configuration_builders",
-        )
+        for attr in ("expectation_configuration_builders",)
```

Comment:
If it's just the one, can we do a simple `assert` instead of using `all`?

Reply:
@cdkini Nice point --done (committed). Thanks!